src/components/IssuePage.vue

```diff
@@ -284,7 +284,14 @@ export default {
       immediate: true,
       handler(pageUid) {
         this.$store.dispatch('issue/LOAD_PAGE', pageUid).then((page) => {
-          this.issue.pages.find(p => p.uid === page.uid).articles = page.articles;
+          const pageIndex = this.issue.pages.findIndex(p => p.uid === page.uid);
+
+          if (pageIndex >= 0) {
+            this.issue.pages[pageIndex].articles = page.articles;
+          } else {
+            console.error('Matteo, check the data please: no pages for this issue');
```

Comment:
if you wanted a neater error message, you could do
`window.app.$store.state.error_message = 'Matteo, check the data please: no pages for this issue';`